backend/services/mgmt/v1alpha1/transformers-service/system_transformers.go

```diff
@@ -628,38 +628,61 @@ var (
 		},
 	}
 
-	systemTransformerSourceMap = map[mgmtv1alpha1.TransformerSource]*mgmtv1alpha1.SystemTransformer{}
+	// base transformers + ee transformers
+	allSystemTransformers = []*mgmtv1alpha1.SystemTransformer{}
+
+	baseSystemTransformerSourceMap = map[mgmtv1alpha1.TransformerSource]*mgmtv1alpha1.SystemTransformer{}
+
+	allSystemTransformersSourceMap = map[mgmtv1alpha1.TransformerSource]*mgmtv1alpha1.SystemTransformer{}
```

Comment:
later would be nice to show all the transformers but have some disabled with like an "upgrade now" button to incentivize users to upgrade in order to get access to those transformers.